### PR TITLE
Update 01-authorization.md

### PR DESCRIPTION
**Pull request by amstee**
ParseWithClaims cannot have nil as a keyFunc

### CODE REVIEW

**Comment by albertoperdomo** [comment]
Hello @amstee,
thanks for submitting this PR.

@alexisluque Can you please review?

If the change makes sense and works, we should also probably update the sample code that goes along with this tutorial. See https://github.com/auth0-samples/auth0-golang-api-samples/blob/master/01-Authorization-RS256/main.go.

**Review by alexisluque — CHANGES_REQUESTED** [review]
Just a typo.

File: articles/quickstart/backend/golang/01-authorization.md [hunk 1]
```diff
@@ -199,15 +199,24 @@ type CustomClaims struct {
 }
 
 func checkScope(scope string, tokenString string) bool {
-	token, _ := jwt.ParseWithClaims(tokenString, &CustomClaims{}, nil)
-
-	claims, _ := token.Claims.(*CustomClaims)
+	token, _ := jwt.ParseWithClaims(tokenString, &CustomClaims{}, func (token *jwt.Token) (interface{}, error) {
+		cert, err := getPermCert(token)
```
Comment (alexisluque) [inline articles/quickstart/backend/golang/01-authorization.md]: This line should be:
`cert, err := getPemCert(token)`

Comment (joshcanhelp) [reply articles/quickstart/backend/golang/01-authorization.md]:
```suggestion
		cert, err := getPemCert(token)
```

**Comment by joshcanhelp** [comment]
@alexisluque - Back from the dead! :) Would you mind blessing this when you have a chance and we'll get this and the sample merged in?

**Review by alexisluque — APPROVED** [review]

**Pull request merged**